File: yelingzi-blog/src/main/java/com/yeling/talk/domian/entity/Talk.java
package com.yeling.talk.domian.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Talk {

    private Integer id;

    private String title;

    private String content;

    private String imageUrl;

    private Integer isTop;

    private Integer state;

    private Integer userId;

    private String nickname;

    private String userAvatar;

    private LocalDateTime createTime;

    private Integer likeCount;

    private Integer commentCount;


}

File: blog/src/types/talk.ts
/**
 * 说说
 */
export interface Talk {
  //说说id
  id: number
  //昵称
  nickname: string
  //用户Id
  userId: number
  //头像
  userAvatar: string
  //标题
  title: string
  //说说内容
  content: string
  //图片列表
  imageUrl: string[]
  //是否置顶 (0否 1是)
  isTop: number
  //点赞量
  likeCount: number
  //评论量
  commentCount: number
  //创建时间
  createTime: string
}

/**
 * 说说
 */
export interface SimpleTalk {
  //说说id
  id: number
  //标题
  title: string
  //说说内容
  content: string
  //图片列表
}
